src/commands/utils/locate/functions/handleCommand.ts

```diff
@@ -0,0 +1,39 @@
+import { ChatInputCommandInteraction, Colors } from 'discord.js';
+import { Panel } from '../panels/panel';
+import { makeEmbed } from '../../../../lib';
+
+const emptyTargetEmbed = makeEmbed({
+    title: 'Locate - Empty target',
+    description: 'Please provide a switch, system or panel that you want to locate.',
+    color: Colors.Red,
+});
+
+const invalidTargetEmbed = makeEmbed({
+    title: 'Locate - Invalid target',
+    description: 'The target you provided is invalid. Please type your search query and choose one from the list.',
+    color: Colors.Red,
+});
+
+const locateEmbed = (panel: Panel) => makeEmbed({
+    title: panel.title,
+    url: panel.docsUrl,
+    description: `Learn more about the ${panel.name} and the flight deck:\n* [${panel.name} Documentation](${panel.docsUrl})\n* [Flight Deck Overview](${panel.flightDeckUrl})`,
+    image: { url: panel.imageUrl },
+    footer: { text: 'Tip: Click the image to view in full size' },
+});
+
+export const handleCommand = async (interaction: ChatInputCommandInteraction<'cached'>, panelMap: Map<string, Panel>) => {
```

Comment:
```suggestion
export async function handleCommand(interaction: ChatInputCommandInteraction<'cached'>, panelMap: Map<string, Panel>) {
```

Existing commands with external functions to the 'main command file' use `export async function` instead of `export const` (I understand it can me more common practice to go with your method, perhaps we can debate it and change existing code and other functions at a later date).